core/src/main/java/hudson/FilePath.java

```diff
@@ -3350,6 +3351,8 @@ public Boolean invoke(@Nonnull File parentFile, @Nonnull VirtualChannel channel)
                     } catch (InvalidPathException e2) {
                         throw new IOException(e2);
                     }
+                } catch(FileSystemException e){
+                    return false;
```

Comment:
add system logging then? Otherwise the issue is just suppressed

Reply:
Warning added with the exception.